packages/app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mudita/mudita-center-app",
-  "version": "2.0.2",
+  "version": "2.0.2-usbdebug.1",
```

Comment:
to be removed, i guess 🤔

Reply:
fixed 9e2d742a8b4d419f433d38b40c08828f004c23bf